tests/sqlparser_snowflake.rs

```diff
@@ -2952,3 +2950,33 @@ fn test_sf_double_dot_notation() {
 
 #[test]
 fn test_parse_double_dot_notation_wrong_position() {}
+
+#[test]
+fn test_table_sample() {
+    snowflake_and_generic()
+        .verified_stmt("SELECT * FROM testtable AS t TABLESAMPLE BERNOULLI (10)");
+
+    // In Snowflake we translate implicit table sample method to bernoulli
+    snowflake().one_statement_parses_to(
+        "SELECT * FROM testtable SAMPLE (10)",
+        "SELECT * FROM testtable TABLESAMPLE BERNOULLI (10)",
```

Comment:
for the scenarios that currently rely on `one_statement_parse_to`, could we represent them faithfully when displaying? e.g this and the `ROW` vs `BERNOULLI` variants etc

Reply:
Not sure if I understand the question, but if you mean whether the variants I added as interchangeable are really interchangeable in the dialect, as far as I understand yes. If not, please elaborate?

Reply:
Ah yeah so I was rather thinking that ideally we preserve the syntax roundtrip, even though they'r interchangeable in some dialects 
```rust
snowflake_and_generic().verified_stmt("SELECT * FROM testtable TABLESAMPLE ROW (20.3)");
snowflake_and_generic().verified_stmt("SELECT * FROM testtable SAMPLE BLOCK (3) SEED (82)");
```